# time of ESKF estimation:
Header header

# a. position:
float64 delta_pos_x_std
float64 delta_pos_y_std
float64 delta_pos_z_std

# b. velocity:
float64 delta_vel_x_std
float64 delta_vel_y_std
float64 delta_vel_z_std

# c. orientation:
float64 delta_ori_x_std
float64 delta_ori_y_std
float64 delta_ori_z_std

# d. gyro. bias:
float64 gyro_bias_x_std
float64 gyro_bias_y_std
float64 gyro_bias_z_std

# e. accel. bias:
float64 accel_bias_x_std
float64 accel_bias_y_std
float64 accel_bias_z_std